### ISSUE TITLE
Predator Concrete Jungle - white lines/divided screen

### ISSUE THREAD

**Issue by PatientWolf69** [PCSX2/pcsx2#2543]
***PCSX2 version:***
v1.5.0-dev-2468-ga0f96189d

***PCSX2 options:***
Everything on default. DX3D11 or OpenGL. HW mode.

***Plugins used:***
None

***Description of the issue:***
Activating the in-game Tech Vision biomask mode causes bright white beams to emanate from every object highlighted by the mode (when DX3D11 renderer is used) or the whole screen is being divided into four parts, each of which seems to generate after-images (when OpenGL renderer is used).

***How to reproduce the issue:***
Run the game in HW mode. Gadget Training mission. Obtain and activate Tech Vision (takes less than a minute to get it).

***Last known version to work:***
I have never encountered a single instance where it worked at least semi-properly other than using SW mode or setting SkipDrawing to 7 (causes the HUD elements to go missing, the image becomes a lot darker, all bloom effects are gone).

***PC specifications:***
System: Windows 10 (64 Bit), CPU: Intel Core I7-7700K, GPU: GTX 1080Ti 

![b](https://user-images.githubusercontent.com/42122850/43690066-509d8f32-9904-11e8-8d35-31d37a7f0638.png)
![c](https://user-images.githubusercontent.com/42122850/43690074-6c205668-9904-11e8-8d8b-18c971c4ab86.png)

**Comment by lightningterror** [comment]
A gs dump will be helpful. On a side note can you try out this PR with the new Skipdraw Offset feature ?
https://ci.appveyor.com/project/gregory38/pcsx2/build/1.4361-master/job/wbdpmua84vif4u4p/artifacts

**Comment by PatientWolf69** [comment]
@lightningterror 
I'm providing the required GS dump below. I used DX3D11 renderer for that. No HW Hacks used. Hopefully, it will help to properly resolve the bug once and for all.

http://www.mediafire.com/file/sk6tvb90yr0m92x/Predator_Concrete_Jungle_-_Tech_Vision_Bug.rar

**Comment by lightningterror** [comment]
The distorted issue on the first image has been fixed, are there any other issues still present ?

**Comment by PatientWolf69** [comment]
I know ;p
I was the one who provided the screenshots from my game on Discord yesterday. Thanks again :)

All of the above visual bugs seem to be gone. No white lines or flickers (D3D11), no divided screen (OpenGL). Also, no invisibility or interface bugs since I don't have to use skip-drawing anymore. Purging D3D9 was definitely a good idea.

The only remaining "issues" that have been there from the beginning are:

a) with D3D11:  a brighter border-like area moving in front of the character in certain vision modes:
![2a](https://user-images.githubusercontent.com/42122850/55746879-3200e300-5a3b-11e9-8e9d-9e8fae1e41be.png)

b) with OpenGL: the border is gone, but certain areas with sky in the background have (almost heat-like) beams radiating from the character (also visible in SW mode)
![1a](https://user-images.githubusercontent.com/42122850/55747150-e8fd5e80-5a3b-11e9-80d6-2b6dff1a70d3.png)

**Comment by lightningterror** [comment]
Dx9 was holding development back by quite a bit, purging it allowed us to port a lot of opengl features to dx11 a lot easier.

**Comment by MrCK1** [comment]
The border in DX looks to be a half-bottom issue. I may be wrong though.

**Comment by PatientWolf69** [comment]
Half-bottom? It moves around with me as the Predator runs.

**Comment by lightningterror** [comment]
@PatientWolf69 can this be retested?

**Comment by PatientWolf69** [comment]
Yes. It's fixed. I have tested it on version 3029-g9332f9b1a back in April 2019 and it's gone. That includes both the white beams (D3D9) and the divided display (OpenGL). No HW Hacks. Thanks again.
Below I enclose the proper screenshots that I have also posted at the Discord Channel.
![Predator_Fixed_Now](https://user-images.githubusercontent.com/42122850/72196328-96603100-3417-11ea-8c54-9d4a1f57ae33.png)
![Predator_Fixed_Now_2](https://user-images.githubusercontent.com/42122850/72196331-9bbd7b80-3417-11ea-91fc-1d0df31aa7fb.png)
